scene/resources/image_texture.h

```diff
@@ -49,15 +49,11 @@ class ImageTexture : public Texture2D {
 
 protected:
 	virtual void reload_from_file() override;
-
-	bool _set(const StringName &p_name, const Variant &p_value);
-	bool _get(const StringName &p_name, Variant &r_ret) const;
-	void _get_property_list(List<PropertyInfo> *p_list) const;
-
 	static void _bind_methods();
 
 public:
 	void set_image(const Ref<Image> &p_image);
+	void _set_image(const Ref<Image> &p_image) { set_image(p_image); }
```

Comment:
This could have a comment above to explain why `set_image()` isn't bound directly.

Reply:
actually, why is `_set_image` needed? set_image is public as well, as long as i can see they are the same picture

Reply:
Using `set_image()` as a setter would hide it as a method. I think I added `_set_image()`, because you can't bind the same method twice.

Reply:
why do you want to have set_image to show up in autocompletion? you are now meant to use the variable, and even if it doesn't show in autocomplete it works, so we still ahve backward compatibility.

Is it about availability from C#?

Reply:
The variable is internal. It doesn't show in the documentation, it only stores data.

Reply:
It used to be non-internal, but causes issued. It was fixed in #61284 (and fixed again in #61795).
It shouldn't be visible in the inspector and we don't really expose properties that are available only in code.